hash_table/hash_table.go

```diff
@@ -0,0 +1,134 @@
+package hash_table
+
+import (
+	"errors"
+)
+
+type Node struct {
+	key   int
+	value int
+	next  *Node
+}
+
+type HashTable struct {
+	arr  []*Node
+	size int
+}
+
+func NewHashTable() *HashTable {
+	return &HashTable{
+		arr: make([]*Node, 10),
+	}
+}
+
+func (h *HashTable) Insert(key int, value int) {
```

Comment:
Missing "insert with existed key" case (we should overwrite the value)

Reply:
@thanks173 nice catch. I suggest to request changes instead of commenting for the next time you find something needs to improve.